### PR TITLE
Disable save until change

### PR DESCRIPTION
**Pull request by toastynerd**
@joshreeves @shenst1 disable save buttons until a change has occurred.

### CODE REVIEW

File: vendor/assets/javascripts/behaviors/changed_form_modal.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ var CFM = Cmsify.ChangedFormModal = function(el) {
   this.$el = $('form');
   if(!this.$el.length) return;
   this.lastData = this.serializeFields();
+  this.detectChange = this.detectChange.bind(this);
+  this.$el.on('keyup', this.detectChange);
+  this.$el.on('click', this.detectChange);
```
Comment (joshreeves) [inline vendor/assets/javascripts/behaviors/changed_form_modal.js]: Is it possible for you to somehow detect the `sortupdate` event here so it's uniform?

File: vendor/assets/javascripts/behaviors/sortable.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ Cmsify.Sortable = function(el, options) {
   this.options = options;
   var sortableElement = sortable(this.$el, this.options);
   sortableElement[0].addEventListener('sortupdate', function(e) {
+    $('input[type=submit]').prop('disabled', false);
```
Comment (joshreeves) [inline vendor/assets/javascripts/behaviors/sortable.js]: ...and then get rid of this.

**Review by joshreeves — COMMENTED** [review]
A question

File: vendor/assets/javascripts/behaviors/sortable.js (hunk 2 above)
Comment (toastynerd) [reply vendor/assets/javascripts/behaviors/sortable.js]: Yeah, I couldn't figure out a way to do it, I tried `'mouseup'`, `'mousedown'` and `'click'` all on `$(window)` but none of them would register until the second change had been made.

File: vendor/assets/javascripts/behaviors/changed_form_modal.js [hunk 3]
```diff
@@ -3,6 +3,12 @@ var CFM = Cmsify.ChangedFormModal = function(el) {
   this.$el = $('form');
   if(!this.$el.length) return;
   this.lastData = this.serializeFields();
+  var detectChange = this.detectChange.bind(this);
+  this.$el.on('keyup', detectChange);
+  this.$el.on('click', detectChange);
+  $('.js-sortable').each(function() {
```
Comment (joshreeves) [inline vendor/assets/javascripts/behaviors/changed_form_modal.js:9]: Add a comment here to look into the event bubbling issue.

**Review by joshreeves — COMMENTED** [review]
Add comment, rebase with master (but squash everything to a single commit), and 🚢

File: vendor/assets/javascripts/behaviors/remote_upload.js [hunk 4]
```diff
@@ -1,6 +1,11 @@
 var remoteUpload = Cmsify.remoteUpload = function(form, elementToClone, elementToInsertBefore, callback) {
   form.dropzone({
     dictDefaultMessage: 'Drop files or click here to upload',
+    maxFiles: 1,
+    maxfilesexceeded: function(file) {
+      this.removeAllFiles();
+      this.addFile(file); 
+    },
```
Comment (joshreeves) [inline vendor/assets/javascripts/behaviors/remote_upload.js:8]: Need to rebase with master. But make sure this PR is squashed to a single commit.

**Pull request merged**